frontend/components/instructionsComponent/navigation/footer/index.tsx

```diff
@@ -53,9 +52,18 @@ const Footer: React.FC = () => {
             >
               SC: {formatAddress(networkConfig.burntPixArchivesAddress)}
             </Text>
-            <Link href={
-              getNetworkConfig(process.env.NEXT_PUBLIC_DEFAULT_NETWORK!).explorerURL + "/address/" + networkConfig.burntPixArchivesAddress
-                } isExternal ml="5px" size="14px" mt="4px">
+            <Link
+              href={
+                getNetworkConfig(process.env.NEXT_PUBLIC_DEFAULT_NETWORK!)
```

Comment:
This may have been missed before, we already have `networkConfig` above so we don't really need to do `getNetworkConfig`